lib/max_subarray.py

```diff
@@ -1,12 +1,25 @@
 
+from json.encoder import INFINITY
```

Comment:
👀 The `INFINITY` symbol is really just an alias for `float('inf')`, which you can access without needing to import anything from the `json` module.